net/tsdial/tsdial.go

```diff
@@ -67,6 +67,23 @@ func (d *Dialer) SetLinkMonitor(mon *monitor.Mon) {
 	d.linkMon = mon
 }
 
+func (d *Dialer) interfaceIndexLocked(ifName string) (index int, ok bool) {
+	if d.linkMon == nil {
+		return 0, false
+	}
+	st := d.linkMon.InterfaceState()
+	tunIf, ok := st.Interface[ifName]
```

Comment:
tunIf is a bit surprising in a generic func. Maybe iface?

Reply:
Done.